Trading/Exchange/bybit/bybit.py

```diff
@@ -0,0 +1,170 @@
+"""
+OctoBot Tentacle
+
+$tentacle_description: {
+    "package_name": "OctoBot-Tentacles",
+    "name": "bybit",
+    "type": "Trading",
+    "subtype": "Exchange",
+    "version": "1.0.0",
+}
+"""
+
+#  Drakkar-Software OctoBot-Tentacles
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import time
+
+from ccxt import BaseError
+
+from octobot_commons.constants import HOURS_TO_SECONDS
+from octobot_trading.constants import CCXT_INFO
+
+from octobot_trading.enums import ExchangeConstantsOrderColumns, ExchangeConstantsPositionColumns, \
+    ExchangeConstantsFundingColumns, PositionStatus, PositionSide
+from octobot_trading.exchanges.types.future_exchange import FutureExchange
+from octobot_trading.exchanges.util.exchange_parsing import set_exchange_value_to_default, \
+    set_exchange_value_if_necessary
+
+
+class Bybit(FutureExchange):
+    DESCRIPTION = ""
+
+    MARK_PRICE_IN_TICKER = True
+    FUNDING_IN_TICKER = True
+
+    BYBIT_SIZE = "size"
+    BYBIT_VALUE = "position_value"
+    BYBIT_MARGIN = "position_margin"
+    BYBIT_LIQUIDATION_PRICE = "liq_price"
+    BYBIT_TIMESTAMP = "created_at"
+    BYBIT_MARK_PRICE = ""
+
+    BYBIT_FUNDING_TIMESTAMP = "funding_rate_timestamp"
+    BYBIT_DEFAULT_FUNDING_TIME = 8 * HOURS_TO_SECONDS
+
+    BYBIT_TIME = "time_now"
+
+    BYBIT_LONG = "Buy"
+
+    @classmethod
+    def get_name(cls):
+        return 'bybit'
+
+    # override to add url
+    def set_sandbox_mode(self, is_sandboxed):
+        if is_sandboxed:
+            self.client.urls['api'] = 'https://testnet.bybit.com'
+        self.client.setSandboxMode(is_sandboxed)
+
+    async def get_symbol_open_positions(self, symbol: str) -> list:
+        return [self.cleanup_position_dict((await self.client.private_get_position_list(
+            {"symbol": self.get_exchange_pair(symbol)}))['result'], PositionStatus.OPEN.value)]
+
+    async def set_symbol_leverage(self, symbol: str, leverage: int):
+        # TODO returned error during implementation tests
```

Comment:
From @GuillaumeDSM in https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/147#discussion_r394684024
> still erroring ?